website/docs/d/iam_policy_document.html.markdown

```diff
@@ -296,4 +202,94 @@ data "aws_iam_policy_document" "override_json_example" {
 }
 ```
 
```

Comment:
It would be helpful to add an example without the `statement` argument:

```hcl
data "aws_iam_policy_document" "source" {
  statement {
    sid = "OverridePlaceholder"
    actions   = ["ec2:DescribeAccountAttributes"]
    resources = ["*"]
  }
}

data "aws_iam_policy_document" "override" {
  statement {
    sid = "OverridePlaceholder"
    actions   = ["s3:GetObject"]
    resources = ["*"]
  }
}

data "aws_iam_policy_document" "yak_politik" {
  source_json = "${data.aws_iam_policy_document.source.json}"
  override_json = "${data.aws_iam_policy_document.override.json}"
}
```

With PR #6052, `aws_iam_policy_document.yak_politik` would evaluate as:
```json
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "OverridePlaceholder",
      "Effect": "Allow",
      "Action": "s3:GetObject",
      "Resource": "*"
    }
  ]
}
```